### PR TITLE
Migrate enums to native graphql enums

### PR DESCRIPTION
**Pull request by cquiroz**

### CODE REVIEW

**Review by toddburnside — COMMENTED** [review]
LGTM

File: common-graphql/src/main/scala/queries/common/UserPreferencesQueriesGQL.scala [hunk 1]
```diff
@@ -123,7 +123,7 @@ object UserPreferencesQueriesGQL {
   @GraphQL
   trait UserPreferencesElevPlotUpdate extends GraphQLOperation[UserPreferencesDB] {
     val document = """
-      mutation userPreferencesUpsert($userId: String = "", $elevationPlotRange: elevation_plot_range = "", $elevationPlotTime: elevation_plot_time = "") {
+      mutation userPreferencesUpsert($userId: String = "", $elevationPlotRange: ExplorePlotRangeEnum = "", $elevationPlotTime: ExplorePlotTimeEnum = "") {
```
Comment (rpiaggio) [inline common-graphql/src/main/scala/queries/common/UserPreferencesQueriesGQL.scala:126]: The default values don't seem like valid enum instances.

Comment (cquiroz) [reply common-graphql/src/main/scala/queries/common/UserPreferencesQueriesGQL.scala:126]: Good point should the parser detect that?

Comment (rpiaggio) [reply common-graphql/src/main/scala/queries/common/UserPreferencesQueriesGQL.scala:126]: That would be nice. For parsing in clue we use grackle and we are "a few" versions behind on it (0.1.9 vs 0.8.2). We should try updating and if that doesn't fix it then see how we can add it to grackle.

Comment (cquiroz) [reply common-graphql/src/main/scala/queries/common/UserPreferencesQueriesGQL.scala:126]: Maybe I should remove the default for now?

**Review by rpiaggio — APPROVED** [review]
LGTM

**Pull request merged**